projects/packages/search/src/dashboard/class-initial-state.php

```diff
@@ -112,6 +114,33 @@ protected function current_user_data() {
 		return $current_user_data;
 	}
 
+	/**
+	 * Gets the post type labels for all of the site's post types (including custom post types)
+	 *
+	 * @return array
+	 */
+	protected function get_post_types_with_labels() {
+
+		$args = array(
+			'public' => true,
+		);
+
+		$post_types_with_labels = array();
+
+		$post_types = get_post_types( $args, 'objects' );
+
+		// We don't need all the additional post_type data, just the slug & label
+		foreach ( $post_types as $post_type ) {
+			$post_type_with_label = array(
+				'slug'  => $post_type->name,
+				'label' => $post_type->label,
+			);
+
+			$post_types_with_labels[] = $post_type_with_label;
```

Comment:
```suggestion
			$post_types_with_labels[ $post_type->name ] = $post_type_with_label;
```
1. Change the return to an associated array